### DISCUSSION TITLE
Carousel map

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#19489] [General]
When I start the map, it spins like a drunk. Had a way to calm down?

**Comment by Dan2077** [comment]
Have you tried the 'use Kalman filter' in any of the profile/ general settings? Suppose to smooth out your compass

**Comment by ZeljkoBG** [comment]
When I turned off the Kalman filter, it behaves normally. I don't know what a Kalman filter is and I've never heard of it. And I've been on navigation for 100 years.

**Reply by Dan2077** [reply]
How is the smoothness of the map rotation when you rotate your phone with and without the Kalman filter? Consider [Issueing a bug report](https://github.com/osmandapp/OsmAnd/issues/new?assignees=&labels=&projects=&template=2-bug-report.yml)

**Comment by ZeljkoBG** [comment]
I feel like you don't understand me. When I launch the app and the app finds the location it spins left and right and so on endlessly. Very annoying. I think the map should face north and stand still. It's only when I'm in motion/simulation the map that rotates in the direction of movement.
